src/lib/scene-internal.ts

```diff
@@ -483,20 +527,31 @@ export class SceneInternal implements Renderer {
         new AttributeMapper({
           maxTextureSize: regl.limits.maxTextureSize,
           desiredSwatchCapacity: attributeMapper.totalSwatches,
-          dataChannelCount: 4,
+          dataChannelCount: 4,  // Must be 4, since these are rendered outputs.
           attributes: [
             {attributeName: 'Hit'},
           ],
         });
 
-    // The instance hit test UVs point to the places in the hit test texture
-    // where the output of the test is written.
-    this.instanceHitTestUvValues =
+    // The instance hit test output UVs point to the places in the hit test
+    // texture where the output of the test is written for each tested sprite.
+    this.instanceHitTestOutputUvValues =
         this.hitTestAttributeMapper.generateInstanceSwatchUvValues();
 
-    // The hitTestValuesFramebuffer is written to by the hit test command and
-    // read from by sampling.
-    this.hitTestValuesFramebuffer = regl.framebuffer({
+    // Just before running a hit test, the specific list of candidate Sprites'
+    // swatch UVs will be copied here, so that when the shader runs, it'll know
+    // where to look for the previous and target values. The output UVs however
+    // do not change. The Nth sprite in the HitTestParameters's sprites array
+    // will always write to the Nth texel of the output framebuffer. Since it's
+    // possible (though unlikely
```

Comment:
incomplete comment

Reply:
Whoops. Yeah, when writing that, I realized that a Sprite that was in the candidates list may not have a swatch assigned (not enough GPU memory). Thinking it through lead me to create the `ActiveIndex` values and buffer, which is described elsewhere. Sentence fragment deleted.